.github/workflows/ci.yml

```diff
@@ -17,7 +17,7 @@ on:
 
 jobs:
   test:
-    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v3
+    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v4.0.0
```

Comment:
it may have come up before, but why are we not pointing at simply v4, so that it will get whatever is the latest v4?

Reply:
There was problem with it because if we release v4 now, dependabot automatically starts updating all the repos to the latest version

https://github.com/fastify/workflows/pull/107